### PR TITLE
Validate application.properties: unknown properties

### PR DESCRIPTION
**Pull request by angelozerr**
Fixes redhat-developer#26

Signed-off-by: azerr <azerr@redhat.com>

### CODE REVIEW

**Comment by angelozerr** [comment]
@xorye you need vscode quarkus PR https://github.com/redhat-developer/vscode-quarkus/pull/68 to play with settings.

File: quarkus.ls/com.redhat.quarkus.ls/src/main/java/com/redhat/quarkus/settings/QuarkusValidationTypeSettings.java [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+		for (DiagnosticSeverity severity : severities) {
+			if (severity.name().toUpperCase().equals(this.severity.toUpperCase())) {
+				return severity;
+			}
+		}
+		return null;
+	}
+
+	/**
+	 * Returns true if the give, property name must be excluded and false otherwise.
```
Comment (xorye) [inline quarkus.ls/com.redhat.quarkus.ls/src/main/java/com/redhat/quarkus/settings/QuarkusValidationTypeSettings.java]: ... if the given property ...

File: quarkus.ls/com.redhat.quarkus.ls/src/main/java/com/redhat/quarkus/settings/QuarkusValidationTypeSettings.java [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+	 * 
+	 * @param propertyName the property name
+	 * @return true if the give, property name must be excluded and false otherwise.
```
Comment (xorye) [inline quarkus.ls/com.redhat.quarkus.ls/src/main/java/com/redhat/quarkus/settings/QuarkusValidationTypeSettings.java]: ... if the given property ...

**Comment by xorye** [comment]
Aside from minor typos, seems good to me

File: quarkus.ls/com.redhat.quarkus.ls/src/main/java/com/redhat/quarkus/settings/QuarkusValidationTypeSettings.java (hunk 1 above)
Comment (angelozerr) [reply quarkus.ls/com.redhat.quarkus.ls/src/main/java/com/redhat/quarkus/settings/QuarkusValidationTypeSettings.java]: @xorye updated

File: quarkus.ls/com.redhat.quarkus.ls/src/main/java/com/redhat/quarkus/settings/QuarkusValidationTypeSettings.java (hunk 2 above)
Comment (angelozerr) [reply quarkus.ls/com.redhat.quarkus.ls/src/main/java/com/redhat/quarkus/settings/QuarkusValidationTypeSettings.java]: @xorye updated

**Comment by angelozerr** [comment]
Thanks @xorye for your review!

**Pull request merged**